TwitchChannelPointsMiner/classes/Twitch.py

```diff
@@ -155,22 +155,22 @@ def get_channel_id(self, streamer_username):
         else:
             return json_response["data"]["user"]["id"]
 
-    def get_followers(self):
-        json_data = copy.deepcopy(GQLOperations.PersonalSections)
-        json_response = self.post_gql_request(json_data)
-        try:
-            if (
-                "data" in json_response
-                and "personalSections" in json_response["data"]
-                and json_response["data"]["personalSections"] != []
-            ):
-                return [
-                    fw["user"]["login"]
-                    for fw in json_response["data"]["personalSections"][0]["items"]
-                    if fw["user"] is not None
-                ]
-        except KeyError:
-            return []
+    def __do_kraken_request(self, query, response_as_json=True):
+        url = f"{API}/{query.strip('/')}"
+        response = self.twitch_login.session.get(url)
+        logger.debug(
+            f"Query: {query}, Status code: {response.status_code}, Content: {response.json()}"
+        )
+        return response.json() if response_as_json is True else response
+
+    def get_followers(self, limit=100, offset=0):
+        followers = []
+        query = f"/kraken/users/{self.twitch_login.get_user_id()}/follows/channels?limit={limit}&direction=asc&sortby=login&offset={offset}"
```

Comment:
IMO the `/kraken` should be moved in `__do_kraken_request`

Reply:
Yes of course like this revision : [https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/tree/0bcd1ac0a05815e4baf947e67038d07921a964bf](url)